src/openproduct/producttypen/urls.py

```diff
@@ -108,6 +108,15 @@
 
 ---
 *Zie de opmerkingen bij de endpoints voor verdere toelichting op specifieke velden.*
+
+
+### Authorisatie
+
+Voor CREATE, UPDATE, PARTIAL_UPDATE en DELETE requests moet user de juiste permissie hebben. Deze permissies zijn in te stellen in de Admin.
```

Comment:
I'd either replace these with the HTTP method names (POST, PUT, etc.) or call it operations instead of requests
```suggestion
Voor schrijfacties (POST, PUT, PATCH, DELETE) moet user de juiste permissie hebben. Deze permissies zijn in te stellen in de Admin.
```